tree/dataframe/src/RDataFrame.cxx

```diff
@@ -755,6 +755,14 @@ processing of these batches. There are no guarantees on the order the batches ar
 order entries of the dataset are processed. Note that this in turn means that, for multi-thread event loops, there is no
 guarantee on the order in which `Snapshot` will _write_ entries: they could be scrambled with respect to the input dataset.
 
+\warning RDataFrame will by default start as many threads as the hardware supports, using up **all** the resources on
+a machine. On a worker node of *e.g.* a batch cluster, this might not be desired if the machine is shared with other
+users. Therefore, **when running on shared computing resources**, use
+```
+ROOT::EnableImplicitMT(i)
+```
+replacing `i` with the number of CPUs/slots that were allocated for this job.
+
```

Comment:
Hi,
I don't think these changes belong in this PR?

Reply:
Thanks for reviewing despite the label `do not review yet`. :-)
It's true that these changes will go somewhere else when I'm done rebasing, but since we are talking about it: Does it make sense to add this to the RDF docs?

Reply:
I was just checking why I was getting all these emails about an RF PR :sweat_smile: 

Yes, it certainly doesn't hurt to add the warning. It should also be repeated in the more general explanation of multi-threading in ROOT, when we have one in that "functional parts" section of the docs.